src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinitionTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class CpsFlowDefinitionTest extends AbstractCpsFlowTest {
+    /**
+     * CpsFlowDefinition's simplest possible test.
+     */
+    @Test
+    public void simplestPossibleTest() throws Exception {
+        CpsFlowDefinition flow = new CpsFlowDefinition(
+                "def sqrt(int x) { return Math.sqrt(x); }\nfor (int i=0; i<10; i++)\nsqrt(i);",
```

Comment:
nit: I personally find it a bit more readable to go ahead and split the string if the scripts requires a newline, something like this:
```suggestion
                "def sqrt(int x) { return Math.sqrt(x); }\n" + 
                "for (int i=0; i<10; i++) { sqrt(i); }",
```